oui/wireless_usb.sh

```diff
@@ -7,5 +7,6 @@ oui_wireless_usb=(
 	00:25:86='TP-LINK Technologies Co., Ltd.|TP Link TL-WN321G USB Adapter'
 	68:1c:a2='Rosewill Inc.|Rosewill RNX-N150HG USB Adapter'
 	6c:fd:b9='Proware Technologies Co Ltd.|Rosewill RNX-N150HG USB Adapter'
+	c8:3a:35='Ralink Technology, Corp. RT5372 Wireless Adapter'
```

Comment:
This doesn't have the right format. Please see https://github.com/EtiennePerot/macchiato/blob/master/README.md#contribute (last paragraph).